envio/config.yaml

```diff
@@ -2,12 +2,11 @@
 name: envio-indexer
 networks:
 - id: 10143
-  # Optional custom RPC configuration - only add if default indexing (Envio HyperSync) has issues
-  # rpc_config:
-  #   url: https://monad-testnet.g.alchemy.com/v2/M4-LB4BW4V2mMSkYaqi-HwhkZkeg302A
-  #   interval_ceiling: 50
-  #   acceleration_additive: 10
-  #   initial_block_interval: 10
+  rpc: # RPC endpoints as fallback providers for added resillience
+  - url: https://testnet-rpc.monad.xyz
+    for: fallback
+  - url: https://monad-testnet.drpc.org
```

Comment:
Hey Denham, thank you for raising a PR! As a small modification, I'd request you to only keep `https://testnet-rpc.monad.xyz` as the fallback RPC URL, as that’s the only official public RPC provided by the Monad Foundation.

Reply:
Got you 👍